lmp/bb-config.sh

```diff
@@ -61,8 +59,6 @@ ACCEPT_EULA_stm32mp1-eval = "1"
 EOFEOF
 else
 	cat << EOFEOF >> conf/local.conf
-ACCEPT_EULA:stm32mp1-disco = "1"
-ACCEPT_EULA:stm32mp1-eval = "1"
```

Comment:
For the changes done at this file I would first like to go over our current factories to see which could be affected (even if dev only, to avoid surprises).

The jobserv.yml change is fine.